tools/wptrunner/wptrunner/environment.py

```diff
@@ -50,6 +51,76 @@ class TestEnvironmentError(Exception):
     pass
 
 
+class _QueueLogger(StructuredLogger):
+    """Structured logger that has a unique name and puts messages on a queue"""
+    def __init__(self, queue):
```

Comment:
```suggestion
    def __init__(self, queue: QueueOfSomeKind):
```

I'm having trouble finding what `queue` is, and this new code seems like a fine place to start using type annotations. I don't know what the actual type is here, but I'll suggest the types where I think it's not obvious.